kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/util/Metrics.java

```diff
@@ -216,6 +211,18 @@ public static MeterProvider<Counter> proxyToServerErrorCounter(String clusterNam
                 clusterName, nodeId);
     }
 
+    public static MeterProvider<Timer> serverToProxyBackpressureTimer(String clusterName, @Nullable Integer nodeId) {
+        return buildTimerMeterProvider(KROXYLICIOUS_SERVER_TO_PROXY_READS_PAUSED_NAME,
+                "Timer showing how long a proxy has paused reading from a upstream connection because the downstream connection is unwriteable",
```

Comment:
```suggestion
                "Timer showing how long the proxy has paused reading from a upstream connection because the downstream connection is unwriteable.",
```